src/main/java/WorkoutViewModel/WorkoutStartViewManager.java

```diff
@@ -0,0 +1,32 @@
+package WorkoutViewModel;
+
+import RoutineManagerModel.Workout;
+
+
+import javax.swing.*;
+import java.awt.*;
+
+import java.util.ArrayList;
+
+public class WorkoutStartViewManager extends JFrame {
+
+    Workout workout0 = new Workout();
+    static ArrayList<String> workoutTest = new ArrayList<String>();
+
+    public void WorkoutStartDisplay(Workout workout, int index){
+        javax.swing.SwingUtilities.invokeLater(new Runnable() {
+            public void run() {
+                WorkoutStartView frame1 = new WorkoutStartView(workout, index);
+            }
+        });
+    }
+//    public static void main(String[] args) {
+//
+//        javax.swing.SwingUtilities.invokeLater(new Runnable() {
+//            public void run() {
+//                WorkoutStartView frame1 = new WorkoutStartView();
+//            }
+//        });
+//
+//    }
+}
```

Comment:
Naming conventions are followed throughout. There are some methods that are comments, like the main function in WorkoutStartViewManager which I believe is required to run your feature. Might be a good idea to add javadoc as the purpose of some of the methods and classes is not very clear.